elasticsearch/reindex/progress.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Usage: [watch -t] ./progress.sh <current index doc count>
+
+DOC_COUNT=$1
```

Comment:
Could we not do:
`OLD_DOC_COUNT=$(curl --silent $ES_URL:9200/$OLD_INDEX/_count | jq ".count")`

and just pass the two index names in?